clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -40,6 +40,9 @@ export const ConsentButtons = ({
 }: ConsentButtonProps) => {
   const isMobile = useMediaQuery("(max-width: 768px)");
   const includeLanguageSelector = i18n.availableLanguages?.length > 1;
+  const includePrivacyPolicyLink =
+    messageExists(i18n, "exp.privacy_policy_link_label") &&
+    messageExists(i18n, "exp.privacy_policy_url");
```

Comment:
This is the same logic used inside the PrivacyPolicyLink component, just surfaced to this parent component for managing css classnames.